packages/env/package.json

```diff
@@ -41,7 +41,7 @@
 		"js-yaml": "^3.13.1",
 		"ora": "^4.0.2",
 		"rimraf": "^3.0.2",
-		"simple-git": "^2.35.0",
+		"simple-git": "^3.5.0",
```

Comment:
https://github.com/steveukx/git-js/releases/tag/simple-git-v3.0.0

It doesn't look like there are breaking changea that could impact the project.

Reply:
As mentioned [here](https://github.com/WordPress/gutenberg/pull/39560#issuecomment-1082312562) I actually tested it against our repo and it works good 👌.

Reply:
We use `simple-git` for example for publishing WordPress packages to npm so you couldn't test it. I wanted to ensure they didn't change any APIs. We will see this Friday 🤣

Reply:
From what I've seen to some extent `wp-env` was also using it for setting things up - so was testing it on this end :)